docker-compose.yml

```diff
@@ -0,0 +1,188 @@
+      image: sail-8.3/app
+      extra_hosts:
+        - 'host.docker.internal:host-gateway'
+      tty: true
+      environment:
+        TZ: ${APP_TIMEZONE}
+        COMPOSER_AUTH: ${COMPOSER_AUTH}
+        XDEBUG_MODE: '${SAIL_XDEBUG_MODE:-off}'
+        XDEBUG_CONFIG: '${SAIL_XDEBUG_CONFIG:-client_host=host.docker.internal}'
+        IGNITION_LOCAL_SITES_PATH: '${PWD}'
+      env_file:
+        - .env
+      volumes:
+        - 'install:/app/_install'
+        - 'storage:/app/storage'
+      networks:
+        - sail
+      depends_on:
+        webapp:
+          condition: service_healthy
+    scheduler:
+      image: sail-8.3/app
+      extra_hosts:
+        - 'host.docker.internal:host-gateway'
+      tty: true
+      env_file:
+        - .env
+      environment:
+        TZ: ${APP_TIMEZONE}
+        COMPOSER_AUTH: ${COMPOSER_AUTH}
+        XDEBUG_MODE: '${SAIL_XDEBUG_MODE:-off}'
+        XDEBUG_CONFIG: '${SAIL_XDEBUG_CONFIG:-client_host=host.docker.internal}'
+        IGNITION_LOCAL_SITES_PATH: '${PWD}'
+      volumes:
+        - 'install:/app/_install'
+        - 'storage:/app/storage'
+      networks:
+        - sail
+      depends_on:
+        webapp:
+          condition: service_healthy
+      command: >
+        sh -c "while [ true ]; do php artisan schedule:run; sleep 60; done"
+    mariadb:
+        image: 'mariadb:11'
+        ports:
+            - '${FORWARD_DB_PORT:-3306}:3306'
+        environment:
+            TZ: ${APP_TIMEZONE}
+            MYSQL_ROOT_PASSWORD: '${DB_PASSWORD}'
+            MYSQL_ROOT_HOST: '%'
+            MYSQL_DATABASE: '${DB_DATABASE}'
+            MYSQL_USER: '${DB_USERNAME}'
+            MYSQL_PASSWORD: '${DB_PASSWORD}'
+            MYSQL_ALLOW_EMPTY_PASSWORD: 'yes'
+        command:
+            --max_allowed_packet=128M
+            --group_concat_max_len=16384
+            --max_connections=200
+        volumes:
+            - 'sail-mariadb:/var/lib/mysql'
+            - './vendor/laravel/sail/database/mariadb/create-testing-database.sh:/docker-entrypoint-initdb.d/10-create-testing-database.sh'
+        networks:
+            - sail
+        healthcheck:
+            test:
+                - CMD
+                - mysqladmin
+                - ping
+                - '-p${DB_PASSWORD}'
+            retries: 3
+            timeout: 5s
+    redis:
+        image: 'redis:alpine'
+        ports:
+            - '${FORWARD_REDIS_PORT:-6379}:6379'
+        environment:
+            TZ: ${APP_TIMEZONE}
+        volumes:
+            - 'sail-redis:/data'
+        networks:
+            - sail
+        healthcheck:
+            test:
+                - CMD
+                - redis-cli
+                - ping
+            retries: 3
+            timeout: 5s
+    mailpit:
+        image: 'axllent/mailpit:latest'
+        ports:
+            - '${FORWARD_MAILPIT_PORT:-1025}:1025'
+            - '${FORWARD_MAILPIT_DASHBOARD_PORT:-8025}:8025'
+        networks:
+            - sail
+#    manticore:
```

Comment:
We need manticore too

Reply:
ok, the container wasn't working that's why I disabled it, i'll see what I can do to repair it

Reply:
> We need manticore too

I've added it back but for now I commented out the manticore.conf that were used, since it make it crash on my conf.

I don't know if it will be a problem ?